hugo/content/memory/_index.md

```diff
@@ -35,69 +33,74 @@ The first 2 bits in a 32-bit address.
+| `UART_RX_STATUS`    | 0xc3000080     | r    | r     | Non-zero when there is data to read.                                                                                                                                                                                  |
+| `UART_RX_DATA`      | 0xc3000084     | r    | r     | Data to read. Only the LSB of the word contains data.                                                                                                                                                                 |
+| `UART_RX_BYTES`     | 0xc3000088     | r    | r     | Number of unread bytes received from client.                                                                                                                                                                          |
+| `UART_TX_STATUS`    | 0xc3000100     | r    | r     | Non-zero when it's OK to write data to send.                                                                                                                                                                          |
+| `UART_TX_DATA`      | 0xc3000104     | w    | w     | Data to send. Only the LSB of the word contains data.                                                                                                                                                                 |
+| `TOUCH_STATUS`      | 0xc4000024     | r    | r     | `TOUCH_STATUS_EVENT_BIT` is 1 when touched. After detecting a touch event (reading a 1), write anything here to acknowledge the event.                                                                                |
+| `QEMU_DEBUG`        | 0xfe00100      | w    | w     | Debug console (only in QEMU). XXX Which byte? LSB?                                                                                                                                                                    |
+| `TK1_NAME0`         | 0xff000000     | r    | r     | ID of core/stick, first part, default "tk1".                                                                                                                                                                          |
+| `TK1_NAME1`         | 0xff000004     | r    | r     | ID of core/stick, second part, default "mkdf".                                                                                                                                                                        |
+| `TK1_VERSION`       | 0xff000008     | r    | r     | Version of core/stick, identitying FPGA bitstream, not the product version or serial. See UDI.                                                                                                                        |
```

Comment:
Do we talk about 'stick' anywhere else in documentation. Could, should we instead call this Tkey FPGA type and version?

Reply:
Good point. I just copied the earlier text.